backend/authentication/serializers/questions_short.py

```diff
@@ -1,12 +1,27 @@
 from rest_framework import serializers
 
-from authentication.models import ProfileAnswer
+from authentication.models import ProfileAnswer, ProfileQuestion
 
 
 class ProfileAnswerShortSerializer(serializers.ModelSerializer):
     question = serializers.StringRelatedField()
+    question_id = serializers.SerializerMethodField()
+
+    def get_question_id(self, answer: ProfileAnswer):
+        return answer.question.id
 
     class Meta:
         model = ProfileAnswer
-        read_only_fields = ("id", "question", "text")
+        read_only_fields = ("id", "question_id", "question", "text")
```

Comment:
Don't we expect to have the `question` id in `question`?

Reply:
Flat is better than nested

Reply:
This is why I did it like this

Reply:
question here is just a string